docs/get-must-not-be-lro.md

```diff
@@ -0,0 +1,70 @@
+# GetMustNotBeLRO
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-14
+
+## Output Message
+
+The Get call cannot be Long Running Operation and it must not have `x-ms-long-running-operation-options` property block.
```

Comment:
The GET operation cannot be Long Running and MUST not have x-ms-long-running-operation property defined?

Reply:
x-ms-long-running-operation this have to be  x-ms-long-running-operation-options propertyblock